mne/viz/evoked.py

```diff
@@ -642,6 +643,14 @@ def plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
         data. Defaults to False.
     show : bool
         Show figure if True.
+    legend : bool
+        If True, automatically create a legend based on evoked.comment.
+    legend_loc : int | string | tuple
+        The location parameter given to the matplotlib legend call. For
+        example, 'upper right'. By default, the location is chosen
+        automatically.
+    legend_fontsize : int
+        The font size for the legend.
```

Comment:
I agree. I think we could also combine the ``legend`` and ``legend_loc`` where ``None`` would be the same as having ``legend=False``.
Also, I think we have a convention of having the ``show`` param being the last one.